axelor-account/src/main/java/com/axelor/apps/account/service/IrrecoverableService.java

```diff
@@ -930,24 +930,16 @@ public Move createIrrecoverableMove(
             invoiceMoveLine.getCredit().multiply(prorataRate).setScale(2, RoundingMode.HALF_UP);
         if (AccountTypeRepository.TYPE_TAX.equals(
             invoiceMoveLine.getAccount().getAccountType().getTechnicalTypeSelect())) {
-          if (invoiceMoveLine.getAccount().getVatSystemSelect() == null
-              || invoiceMoveLine.getAccount().getVatSystemSelect() == 0) {
+          if (invoiceMoveLine.getVatSystemSelect() == null
+              || invoiceMoveLine.getVatSystemSelect() == 0) {
             throw new AxelorException(
-                TraceBackRepository.CATEGORY_CONFIGURATION_ERROR,
-                I18n.get(AccountExceptionMessage.MISSING_VAT_SYSTEM_ON_ACCOUNT),
-                invoiceMoveLine.getAccount().getCode());
+                TraceBackRepository.CATEGORY_CONFIGURATION_ERROR, "No VAT system");
```

Comment:
Don't put this message like this, put it in a variable in AccountExceptionMessage
And translate it